.codecov.yml

```diff
@@ -5,3 +5,8 @@ codecov:
   # DAutoTester and auto-tester are not in the default list
   ci:
     - !travis
+
+coverage:
+  fixes:
+    - "test/.*/::src/"
+    - "fail_compilation/::src/"
```

Comment:
This shouldn't be necessary - in fact everything in `extra_files` can be safely ignored as they are stored `.lst` coverage files which are used to test the coverage counting.